Intersect.Server/Entities/Events/CommandProcessing.cs

```diff
@@ -419,13 +419,75 @@ Stack<CommandInstance> callStack
         )
         {
             var success = false;
-            if (command.Add) //Try to give item
+
```

Comment:
@jcsnider You get first pass at this file.

Reply:
This bit is now completely different.